scripts/monorepo/compass-deps-list.js

```diff
@@ -0,0 +1,400 @@
+    description: 'The query history sidebar.',
+    repository: 'https://github.com/mongodb-js/compass-query-history.git'
+  },
+  {
+    name: '@mongodb-js/compass-schema',
+    description: 'Compass Schema Tab Plugin',
+    repository: 'https://github.com/mongodb-js/compass-schema.git'
+  },
+  {
+    name: '@mongodb-js/compass-schema-validation',
+    description: 'Compass plugin for document JSON schema validation',
+    repository: 'https://github.com/mongodb-js/compass-schema-validation.git'
+  },
+  {
+    name: '@mongodb-js/compass-server-version',
+    description: 'Compass Server Version',
+    repository: 'https://github.com/mongodb-js/compass-server-version.git'
+  },
+  {
+    name: '@mongodb-js/compass-serverstats',
+    description: 'Compass Real Time Server Stats Component.',
+    repository: 'https://github.com/mongodb-js/compass-serverstats.git'
+  },
+  {
+    name: '@mongodb-js/compass-shell',
+    description: 'Compass Shell Plugin',
+    repository: 'https://github.com/mongodb-js/mongosh.git',
+    // Because we don't want to get the whole history of mongosh and there is no
+    // way to extract it in a better way
+    importStrategy: 'copy'
+  },
+  {
+    name: '@mongodb-js/compass-sidebar',
+    description: 'Sidebar external plugin',
+    repository: 'https://github.com/mongodb-js/compass-sidebar.git'
+  },
+  {
+    name: '@mongodb-js/compass-ssh-tunnel-status',
+    description: 'Compass SSH Tunnel Status',
+    repository: 'https://github.com/mongodb-js/compass-ssh-tunnel-status.git'
+  },
+  {
+    name: '@mongodb-js/compass-status',
+    description: 'Compass Status Plugin',
+    repository: 'https://github.com/mongodb-js/compass-status.git'
+  },
+  {
+    name: '@mongodb-js/dl-center',
+    description: 'mongodb download center internal tools',
+    repository: 'https://github.com/mongodb-js/dl-center.git'
+  },
+  {
+    name: '@mongodb-js/electron-wix-msi',
+    description: 'Creates an MSI installer for your Electron app',
+    repository: 'https://github.com/mongodb-js/electron-wix-msi.git'
+  },
+  {
+    name: '@mongodb-js/triejs',
+    description: 'Customizable trie data structure built in JavaScript.',
+    repository: 'https://github.com/mongodb-js/triejs.git'
+  },
+  {
+    name: 'app-migrations',
+    description: 'Helper for application schema migrations.',
+    repository: 'https://github.com/mongodb-js/app-migrations.git'
+  },
+  {
+    name: 'compass-preferences-model',
+    description: 'Compass preferences model.',
+    repository: 'https://github.com/mongodb-js/compass-preferences-model.git'
+  },
+  {
+    name: 'compass-user-model',
+    description: 'MongoDB user model.',
+    repository: 'https://github.com/mongodb-js/compass-user-model.git'
+  },
+  {
+    name: 'detect-coordinates',
+    description: 'Geo coordinates detection based of a mongodb-schema type.',
+    repository: 'https://github.com/mongodb-js/detect-coordinates.git'
+  },
+  {
+    name: 'electron-license',
```

Comment:
Do we need this one too as part of the electron stuff? https://github.com/mongodb-js/electron-wix-msi

Reply:
So this one is included in the list, just under a different name: `@mongodb-js/electron-wix-msi`